feat(develop): Update Response Context (#14498)
getsentry/sentry-docs: develop-docs/sdk/data-model/event-payloads/contexts.mdx

Response context contains information about the HTTP response associated with the event.

This is mostly set on transactions in a web server environment where one transaction represents a HTTP request/response cycle.

`cookies`

> : _Optional_. The cookie values. Can be given unparsed as string, as dictionary, or as a list of tuples.

Reviewer: ```suggestion
: _Optional_. The cookie values. Can be passed, unparsed, as a string, as a dictionary, or as a list of tuples.
```

- Example: `["PHPSESSID=12345; csrftoken=1234567"]`

`headers`

> : _Optional_. A dictionary of submitted headers. If a header appears multiple times it, needs to be merged according to the HTTP standard for header merging. Header names are treated case-insensitively by Sentry.

Reviewer: ```suggestion
: _Optional_. A dictionary of submitted headers. If a header appears multiple times it, needs to be merged according to the HTTP standard for header merging. Sentry treats header names as case-insensitive.
```

> - Example: `[["Content-Type", "text/plain"]]`

Reviewer 2: it says this is a dictionary but the example is a list.

`status_code`

: **Required.** The integer status code from the HTTP response associated with the event.

- Example: `200`

`body_size`

: _Optional_. HTTP response body size in bytes.

- Example: `1000`

`data`

> : _Optional_. Response data in any format that makes sense. SDKs should discard large and binary bodies by default. Can be given as a string or structural data of any format.

Reviewer 2: can this really be everything? is EAP not limited to only basic types like str, int, float, bool, null and lists of those types?
